app/uk/gov/hmrc/mtdsatestsupportapi/controllers/requestParsers/validators/CreateBusinessValidator.scala

```diff
@@ -48,7 +48,9 @@ class CreateBusinessValidator @Inject() (clock: Clock) extends Validator[CreateB
   private def bodyEnumValidation(body: JsValue): Either[List[MtdError], Unit] = {
```

Comment:
Is this the most suitable place for the tax year validations here? They are not enums so takes the purpose of the method name away a little bit. Would they be better placed under `bodyFieldValidation`?

Reply:
Yes. Good spot! Will rename to something more suitable.